app/control.c

```diff
@@ -743,28 +789,36 @@ control_init(int32_t socket_id)
 	netl_h->cb.route6 = route6;
 	netl_h->cb.link = eth_link;
 
-	struct ether_addr invalid_mac = {{0x00, 0x00, 0x00, 0x00, 0x00, 0x00}};
-	struct in_addr invalid_ip = {INADDR_ANY};
-	struct in6_addr invalid_ip6 = IN6ADDR_ANY_INIT;
-	uint8_t nexthop_id;
-	if (neighbor4_add_nexthop(neighbor4_struct[socket_id], &invalid_ip,
-				  &nexthop_id, NEI_ACTION_DROP) < 0) {
+	struct in_addr invalid_ip = {INADDR_ANY},
```

Comment:
We cannot use the same one, it is used to send packets to the KNI, otherwise blackholed packets are also forwarded to the KNI (BAD_PORT is checked in ip_process). We may rewrite how it is handled though.